spec/system/question_answer_spec.rb

```diff
@@ -0,0 +1,148 @@
+require 'rails_helper'
+
+RSpec.describe '質問から回答までの一連の流れ', type: :system do
+  describe '基本的な操作' do
+    let!(:user) { create(:user)}
+    it '質問から回答までの一連の流れが正しく行われること' do
```

Comment:
たぶん現場であまりそこまで指摘されることないかもしれないですが、こうしたい気持ちはあります笑
（細かい性格ですみません 汗）

```suggestion
    it '質問・質問更新・回答・回答削除・解決済にするまでの一連の流れが正しく行われること' do
```

とりあえず、テストは書くことが正義なので、書いた自分をまず褒めることが大事ですが、it を細かく区切ってみてもいいかもしれないです！